kuma/wiki/utils.py

```diff
@@ -59,3 +67,42 @@ def tidy_content(content):
         return tidied.decode('utf-8'), errors
     else:
         return content
+
+
+def analytics_user_counts(*revs):
+    """Given some document revision numbers, returns a dict matching those
+    with the number of users Google Analytics thinks has visited each revision.
+
+    """
+
+    scopes = ['https://www.googleapis.com/auth/analytics.readonly']
+    credentials = ServiceAccountCredentials.from_json_keyfile_dict(
+        json.loads(config.GOOGLE_ANALYTICS_CREDENTIALS), scopes=scopes)
```

Comment:
I'm inclined, now that you mention it, to make both case 1 and case 2 raise an ImproperlyConfigured error with different messages.